code/game/gamemodes/wizard/artefact.dm

```diff
@@ -708,6 +708,32 @@ var/global/list/multiverse = list()
 			H.equip_to_slot_or_del(new /obj/item/weapon/shield/riot/roman(H), slot_l_hand)
 			H.equip_to_slot_or_del(new /obj/item/weapon/twohanded/spear(H), slot_back)
 
+/obj/item/device/necromantic_stone/proc/spawnheresy(mob/living/carbon/human/H as mob)
+	H.set_species("Human")
+	if (H.gender == MALE)
+		H.gender = FEMALE
+	H.h_style = random_hair_style(H.gender, H.species)
+
+	if(prob(50))
+		H.change_hair_color(216, 192, 120)
+	else
+		H.change_hair_color(140, 170, 74)
+
+	H.update_dna()
+	H.update_body()
+
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(H), slot_shoes)
+	H.equip_to_slot_or_del(new /obj/item/clothing/head/kitty(H), slot_head)
+	H.equip_to_slot_or_del(new /obj/item/clothing/under/schoolgirl(H), slot_w_uniform)
+	H.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest(H),  slot_wear_suit)
+	H.equip_to_slot_or_del(new /obj/item/weapon/katana(H), slot_r_hand)
+	H.equip_to_slot_or_del(new /obj/item/weapon/shield/riot/roman(H), slot_l_hand)
```

Comment:
Energy katanas waste you in like two hits, it's so nuts haha

I think it'd be TOO strong.